addon/modifiers/au-floating-ui.js

```diff
@@ -0,0 +1,158 @@
+import Modifier from 'ember-modifier';
+import { assert } from '@ember/debug';
+import { registerDestructor } from '@ember/destroyable';
+
+import {
+  autoUpdate,
+  computePosition,
+  flip,
+  hide,
+  offset,
+  arrow,
+} from '@floating-ui/dom';
+
+import { merge } from 'merge-anything';
+
+export default class AuFloatingUiModifier extends Modifier {
+  modify(
+    floatingElement,
+    [_referenceElement, _arrowElement],
+    { defaultPlacement = 'bottom-start', options = {}, middleware = [] }
```

Comment:
I'm not entirely sure that using `@options` and `@middleware` like this is the correct way to do it?

Regarding `@options`: is it better to split up the individual values to something like `@floaterOffset`, `@arrowOffset`, etc?
Regarding `@middleware`: should there be this much control over the configuration of the actual floating-ui component?